### ISSUE TITLE
VoiceOver & iOS 13 voice control support

### ISSUE THREAD

**Issue by abotkin** [abotkin/spacetrader#6]
The beauty of this game is that it doesn't rely on graphics but rather textual descriptions & user choices given options.  It would make a lot of sense if this game could be played using VoiceOver effectively.  In addition, with iOS 13 having increased voice control support, it should be possible for a user to be able to play the game simply using their voice too.
